Formula/mmseqs2.rb

```diff
@@ -0,0 +1,46 @@
+class Mmseqs2 < Formula
+  desc "Software suite for very fast protein sequence search and clustering"
+  homepage "https://mmseqs.org/"
+  url "https://github.com/soedinglab/MMseqs2/archive/1-c7a89.tar.gz"
+  version "1-c7a89"
+  sha256 "e756a0e5cb3aa8e1e5a5b834a58ae955d9594be1806f0f32800427c55f3a45d5"
+
+  depends_on "cmake" => :build
+  depends_on "gcc"
+  cxxstdlib_check :skip
+
+  fails_with :clang # needs OpenMP support
+
+  resource "documentation" do
+    url "https://github.com/soedinglab/MMseqs2.wiki.git",
+        :revision => "6dbd3666edb64fc71173ee714014e88c1ebe2dfc"
```

Comment:
should we bump this to 8055a1bf70bb3923e36ba243d5c582fced4896fa?

Reply:
With the next release, there is already some new stuff there.